pkg/daemon/daemon.go

```diff
@@ -44,43 +45,41 @@ type daemon struct {
 // NewDaemon initializes the need components including k8s client, subnet manager client plugins, and guid pool.
 // It returns error in case of failure.
 func NewDaemon() (Daemon, error) {
-	glog.Info("daemon NewDaemon():")
+	log.Info().Msg("daemon NewDaemon():")
```

Comment:
There should not be method names in non debug logs

Reply:
agree